clang/lib/Format/CMakeLists.txt

```diff
@@ -53,3 +53,10 @@ foreach (file IN LISTS files)
 endforeach ()
 
 add_custom_target(clang-format-check-format DEPENDS ${check_format_depends})
+
+add_custom_target(clang-format-style-options
+    COMMAND "${Python3_EXECUTABLE}" dump_format_style.py
```

Comment:
It seems that the command always runs even if the header files are not outdated.